src/views/blog-post/article-revisions/article-revisions.module.scss

```
@@ -0,0 +1,173 @@
+@use "src/tokens/index" as *;
+
+:root {
+	--date-dropdown_padding-vertical: var(--spc-1x);
+	--date-dropdown_padding-horizontal: var(--spc-2x);
+	--date-dropdown_gap: var(--spc-1x);
+	--date-dropdown_corner-radius: var(--corner-radius_s);
+
+	--date-dropdown_arrow-size: 1rem;
+
+	--date-dropdown_foreground-color: var(--secondary_on-variant);
+	--date-dropdown_tag-color: var(--secondary_default);
+	--date-dropdown_background-color: var(--background_secondary);
+	--date-dropdown_background-color_focused: var(--background_focus);
+	--date-dropdown_state_hovered: var(--surface_secondary_emphasis-low);
+	--date-dropdown_state_pressed: var(--surface_secondary_emphasis-high);
+
+	--date-dropdown_border-width: var(--border-width_s);
+	--date-dropdown_border-color: var(--secondary_variant);
+	--date-dropdown_border-color_hovered: var(--secondary);
+	--date-dropdown_border-color_pressed: var(--secondary);
+
+	--date-dropdown_focus-outline_width: var(--border-width_focus);
+	--date-dropdown_focus-outline_color: var(--focus-outline_secondary);
+
+	--popup_min-width: 240px;
+	--popup_popover_version_foregrouund_color: #21333f80;
+
+	// POPUP STYLES ==> taken from src/components/select/select.module.scss
+	--popup_corner-radius: var(--corner-radius_m);
+	--popup_padding: var(--spc-2x);
+	--popup_background-color: var(--background_primary);
+	--popup_item_background-color_selected: var(--surface_primary_emphasis-low);
+
+	--popup_item_padding: var(--spc-2x);
+	--popup_item_label_padding: var(--spc-1x);
+	--popup_item_min-height: var(--min-target-size_m);
+	--popup_item_corner-radius: calc(
+		var(--popup_corner-radius) - var(--popup_padding)
+	);
+	--popup_item_label_color: var(--foreground_emphasis-high);
+	--popup_item_version_color: var(--foreground_disabled);
+	--popup_item_version_selected_color: var(--foreground_emphasis-medium);
+	--popup_item_background-color_hovered: var(--surface_primary_emphasis-medium);
+}
+
+.button {
+	padding: var(--date-dropdown_padding-vertical)
+		var(--date-dropdown_padding-horizontal);
+	background-color: var(--date-dropdown_background-color);
+	border: var(--date-dropdown_border-width) solid
+		var(--date-dropdown_border-color);
+	border-radius: var(--date-dropdown_corner-radius);
+	display: flex;
+	gap: var(--date-dropdown_gap);
+	align-items: center;
+	font-family: var(--pfp-font-family);
+	font-weight: var(--weight_semibold);
+	font-size: var(--button_regular_font-size);
+	line-height: var(--button_regular_line-height);
+	@include transition(background-color);
+	@include transition(border);
```

```suggestion
	@include transition(background-color border);
```
These styles override each other - can place them both in the same statement.

---

Done